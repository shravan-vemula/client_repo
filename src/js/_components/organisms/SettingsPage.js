import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import { AppBar, Toolbar } from "@material-ui/core";
import AvatarIcon from "../atoms/AvatarIcon";
import Grid from "@material-ui/core/Grid";
import SettingsSideBar from "js/_components/organisms/SettingsSideBar";
import SettingsHeader from "js/_components/organisms/SettingsHeader";

const useStyles = makeStyles((theme) => ({
  root: {
    flexGrow: 1,
  },

  base: {
    boxShadow: "0 2px 0 0 #eaedf3",
    backgroundColor: "#ffffff",
  },
  Alignment: {
    textAlign: "left",
  },
}));

export default function SettingsPage() {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <Grid container>
        <AppBar position="fixed" color="inherit">
          <Toolbar>
            <SettingsHeader />
            <AvatarIcon props={{ spacing: 50 }} />
          </Toolbar>
        </AppBar>
        <div>
          <SettingsSideBar />
        </div>
      </Grid>
    </div>
  );
}
