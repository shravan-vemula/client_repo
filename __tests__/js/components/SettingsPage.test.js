import React from "react";
import { cleanup } from "@testing-library/react";
import SettingsPage from "js/_components/organisms/SettingsPage";

afterEach(cleanup);

describe("SettingsPage Component", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<SettingsPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
