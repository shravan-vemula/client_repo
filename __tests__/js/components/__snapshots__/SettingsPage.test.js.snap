// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsPage Component renders correctly 1`] = `ShallowWrapper {}`;
